### PR TITLE
kk/test-coverage

### PR DESCRIPTION
**Pull request by emu47**
some basic test coverage thoughts.

### CODE REVIEW

**Comment by rhernand3z** [comment]
I would add that each developer MUST document how to run tests on their Service/App. It can be documented in the repository ReadMe or WIKI. There are various flavors of unit testing frameworks out there depending on the language, clear instructions on how to execute tests will help other developers add/improve/verify test coverage when collaborating in a project.

CC: @kfriedman

File: standards/test-coverage.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Test Coverage
+
+Apps SHOULD enforce a sane testing strategy. Some basic guidelines follow. Developers are encouraged to add and expand on this list. 
+
+## Unit Testing
+
+Unit test coverage SHOULD cover at least 70% of the codebase. 
+
+If test coverage remains under 70%, each subsequent pull requests MUST at least nominally increase the overall coverage.
```
Comment (kfriedman) [inline standards/test-coverage.md]: I really like this!

**Comment by kfriedman** [comment]
One more thought: should we consider adding a section on CI?

The team has started to use Travis more and more.

Maybe this belongs in a different/separate standard. It might also be too early to make a recommendation.

Thoughts?

**Comment by emu47** [comment]
Yeah, that sounds good, but maybe open an issue about adding CI / Travis information? I haven't yet started using it, so I don't know if I'd be the write person to write anything about it at present.

**Comment by kfriedman** [comment]
Good call @emu47. I'll open an issue for a future enhancement.

**Comment by kfriedman** [comment]
Should we merge this v1?

**Pull request merged**